Trainer.opy

```diff
@@ -2510,9 +2510,6 @@ def menu_speed_set_text_and_color():
     elif eventPlayer.menu_opt_value == 175:
         eventPlayer.menu_opt_var[eventPlayer.menu_index] = opt_array_speed[1][5]
         menu_on_opt_color[eventPlayer.menu_index] = opt_array_speed[2][5]
-    else:
-        eventPlayer.menu_opt_var[eventPlayer.menu_index] = "{0} %".format(eventPlayer.menu_opt_value)
-        menu_on_opt_color[eventPlayer.menu_index] = default_menu_on_index_color

```

Comment:
I am not sure what this code is used for. If it remains it overwrites the variable "speed_current".

Reply:
hmmm, I'm not sure either. I think the 175 reperesents the in-game speed boost cap. I think at one version of the code it had all the possible speed boost combinations and this may be a remnant of that?